src/test/java/tests/RepositoryPickerStateTests.java

```diff
@@ -0,0 +1,100 @@
+package tests;
+
+import org.junit.Test;
+import ui.components.pickers.PickerRepository;
+import ui.components.pickers.RepositoryPickerState;
+
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class RepositoryPickerStateTests {
+    @Test
+    public void constructor_constructRepoPickerState_oneRepoSelected() {
+        Set<String> existingRepositories = new HashSet<>(Arrays.asList("A/A", "B/B"));
```

Comment:
Don't use test data that are not typical. One letter org/repo names are not typical.